LFS171x/indy-material/nodejs/AgentDemoScript.md

```diff
@@ -0,0 +1,319 @@
+
+*   Each has no relationships or messages
+*   Each has one credential - from the Government - that has the official "Name" of the Identity (Alice, Faber, etc.)
+*   Each has some things under Proof Requests and Issuing. We'll get into those later.
+
+|![AgentScreen](images/AgentScreen.png "Agent Home Screen from Demo")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+
+You will also see that at the bottom of the screen each user has a DID. Each generated the DID and published the DID to the Hyperledger Indy registry. Knowing a DID, another Agent can look up on the Ledger and get back the public key and endpoint associated with that DID.  We'll use those DIDs later to connect the Agents so that they can exchange messages.
+
+> **NOTE:** From time to time in running the demo you must refresh the Browser (Crl-R or Cmd-R) to update the screen data. This is needed when an action by one Identity (e.g. Alice) affects the data of another (e.g. Faber).
+
+
+### Step 2: Setting up Faber
+
+Before we get started, Faber has to do some one time setup that we'll do first.  Go to the Faber browser tab (localhost:3002) and click **Issuing**. We're going to do two things here:
+
+
+
+*   First, we'll post to the Hyperledger Indy Ledger a transcript _Schema_. This is a public definition of what fields (claims) are in Transcript Credentials that Faber will issue to its graduates. Since it is public, the Schema could be published by anyone (such as the "Association of Colleges and Universities") and can be used by anyone (such as other institutions like "Port Chester University").
+*   Second, we'll post to the Hyperledger Indy Ledger a "Credential Definition," an Indy component that links a particular Credential Issuer (Faber in this case), with a Schema (the Transcript Schema), an optional Revocation Registry (we're not going to use that) and a series of Keys used for encryption and signing of the Claims in the Credentials.
+
+Together, that public data is what allows Faber to issue a Transcript Credential to Alice that she can later prove to Acme.
+
+Back to our story.  Faber has just set up a Hyperledger Indy Agent that can issue credentials. John Academic, Faber's most awesome transcript keeper, needs to get things working. He will have to write a Schema and Credential Definition to the Indy Ledger to do this. It is a bit complicated but nothing John Academic can't handle!
+
+
+
+1.  If you haven't already done so, go to the Faber agent tab and click **Issuing**.
+
+|![CreateSchema](images/CreateSchema.png "Publish schema to the Indy ledger")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+1.  Give the schema a name, a version and a list of attributes. For example, _Transcript 1.3 with name, degree, status, year, average and social security number._

```

Comment:
This should be a continuation of the previous list. Need to indent the above picture to continue the list.

Reply:
Fixed - thanks for the tip!